pkg/ioutils/fswriters.go

```diff
@@ -109,6 +117,12 @@ func (w *atomicFileWriter) Close() (retErr error) {
 			return err
 		}
 	}
+
+	info, statErr := w.f.Stat()
+	if statErr == nil {
```

Comment:
Why wouldn’t this be a fatal error?

Reply:
That is a tricky question, 99.9% of all callers don't need that Mod time so might not care about this failure. On the other hand, if this actually fails then something pretty weird is going an and maybe you still want to know.

Reply:
Is fatal now

Reply:
AFAICS this error is still ignored.

Reply:
I’m sorry, never mind, my mistake. It’s even very clearly documented.